server/test/State.test.ts

```diff
@@ -820,6 +820,14 @@ describe("State", () => {
       //--- ends state.distributeRoles()
     });
 
+    describe("for not a werewolf, mason, minion or doppelganger", () => {
+      it("should have empty nigttime messaging", () => {
```

Comment:
what is "messaging" here? these might be coordinated roles so are we allowing communication? or is it nighttime instructions? guess i'm just trying to figure out how to read this part

Reply:
Changed those labels slightly, but these tests are verifying the functionality of `State.nighttimeMessage` which is the messages included on the Night Screen for:
- what role the Doppelganger became
- who, if any, the werewolves are for the minion, or other werewolves for a werewolf
- who the other mason is

Reply:
right, the server message